### ISSUE TITLE
Is there a way to adjust the frame rate like the c language?

### ISSUE THREAD

**Issue by Choikyungho9** [ibrahimqazi/TFmini-Plus-LiDAR-interfacing-with-Raspberr-Pi#1]
Is there a way to adjust the frame rate like the c language?

Arduino can adjust the frame rate to 1000 Hz, but I would like to know if it is possible with Python.

**Comment by ibrahimqazi** [comment]
Yes it is also possible with Python, you can send commands in the same way as Arduino.

**Comment by Choikyungho9** [comment]
Thank you. 
So.. how can I change the frame rate in your code?
